src/aiida/engine/processes/calcjobs/tasks.py

```diff
@@ -253,7 +253,6 @@ async def task_monitor_job(
     async def do_monitor():
         with transport_queue.request_transport(authinfo) as request:
             transport = await cancellable.with_interrupt(request)
-            transport.chdir(node.get_remote_workdir())
```

Comment:
Same here

Reply:
yup, I had moved that to the only place it was used: `always_kill()`